pkg/agent/openflow/pipeline.go

```diff
@@ -191,6 +191,15 @@ var (
 	priorityDNSIntercept    = uint16(64991)
 	priorityDNSBypass       = uint16(64992)
 
+	// priorityAdd is used to get a priority by adding an offset to a defined priority. Note that, for flows of non-NetworkPolicy,
+	// only when all existing priorities (priorityHigh, priorityNormal, priorityLow and priorityMiss) cannot meet the design
```

Comment:
better to organize this sentance as below:
```
// priorityAdd should be used only when all predefined priorities (priorityHigh, priorityNormal, priorityLow and
// priorityMiss) cannot meet the design requirement and a new priority has to be introduced.
```

Reply:
Thanks, much better.